indra/newview/scripts/lua/require/LLInventory.lua

```diff
@@ -0,0 +1,48 @@
+local leap = require 'leap'
+local mapargs = require 'mapargs'
+
+local LLInventory = {}
+
+-- Get the items/folders info by provided IDs,
+-- reply will contain "items" and "categories" tables accordingly
+function LLInventory.getItemsInfo(items_id)
```

Comment:
Since this parameter is meant to be an array table of IDs, may I suggest parameter name `item_ids` rather than `items_id`?